clients/imodels-client-management/src/base/internal/AxiosRetryPolicy.ts

```diff
@@ -0,0 +1,39 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { isAxiosError } from "axios";
+import { Constants } from "../../Constants";
+import { GetSleepDurationInMsParams, HttpRequestRetryPolicy, ShouldRetryParams } from "../types";
+
+import { BackoffAlgorithm } from "./ExponentialBackoffAlgorithm";
+
+/** Default implementation for {@link HttpRequestRetryPolicy}. */
+export class AxiosRetryPolicy implements HttpRequestRetryPolicy {
+  private readonly _backoffAlgorithm: BackoffAlgorithm;
+
+  public constructor(
+    backoffAlgorithm: BackoffAlgorithm,
+    maxRetries: number = 3
```

Comment:
I think it would be more readable if we specified all constants when constructing these and `ExponentialBackoffAlgorithm` params in `fillManagementClientConfiguration`/`fillAuthoringClientConfiguration`. then these functions will be one place to see what are the defaults for client behavior,

Reply:
updated